manifests/0000_05_config-operator_02_config.clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
+  name: system:openshift:aggregate-to-cluster-reader
```

Comment:
Since the rename is outstanding, how about one like https://github.com/openshift/cloud-credential-operator/pull/59/files#diff-48fe86d9a241fe06f79289df1b6c1ee8R4

`name: system:openshift:cluster-config-operator:cluster-reader`

Reply:
fixed! thanks.